test/drivers/test_fluke_8846A.py

```diff
@@ -316,6 +316,78 @@ def test_measurement_diode(funcgen, dmm, rm):
     assert meas == pytest.approx(TEST_DIODE, abs=TEST_DIODE_TOL)
 
 
+@pytest.mark.drivertest
+def test_get_nplc(dmm):
+    query = dmm.get_nplc()
+    assert query == pytest.approx(10)
+
+
+@pytest.mark.drivertest
+def test_set_nplc(dmm):
+    dmm.voltage_dc()
```

Comment:
You can use "@pytest.mark.parametrize" to inject a range of different test data into the function. That will help you test a range of different modes.

Reply:
I've parameterized the tests as of f657fa2